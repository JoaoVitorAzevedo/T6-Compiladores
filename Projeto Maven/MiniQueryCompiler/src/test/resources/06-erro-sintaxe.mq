LOAD "data.json"
FILTER .idade >