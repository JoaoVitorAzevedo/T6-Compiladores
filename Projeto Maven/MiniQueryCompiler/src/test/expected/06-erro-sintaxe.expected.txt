ERRO DE SINTAXE: Linha 2:15 - mismatched input '<EOF>' expecting {'.', STRING, NUMBER}